src/Microsoft.OData.Core/Json/ODataJsonWriterUtils.cs

```diff
@@ -505,10 +529,6 @@ private static void ODataResourceValueToString(StringBuilder sb, ODataResourceVa
                     await jsonWriter.WriteValueAsync(detail.Target).ConfigureAwait(false);
                 }
 
-                // "message": "$search query option not supported",
-                await jsonWriter.WriteNameAsync(JsonConstants.ODataErrorMessageName).ConfigureAwait(false);
-                await jsonWriter.WriteValueAsync(detail.Message ?? string.Empty).ConfigureAwait(false);
-
                 // }

```

Comment:
What's up with this commented out brace?

Reply:
I believe that is logic that I only moved. I'll check and remove if it's serving any purpose

Reply:
@habbes argh! This is not an isolated case. The original author added the comment to document which character is being written. Do you think we should remove them? I think we should...

Reply:
Ooh. I see. Well if the comment is still accurate, you could make that call. I didn't know it was documenting the intended output.